src/app/generate_zkapp_txn/generate_zkapp_txn.ml

```diff
@@ -0,0 +1,35 @@
+open Core
+open Mina_base
+open Mina_signature_kind_type
+
+let () =
+  let txn =
+    let txn = Lazy.force (Zkapp_command.dummy ~signature_kind:Testnet) in
+    { txn with
+      account_updates =
+        Zkapp_command.Call_forest.forget_hashes
+        @@ Zkapp_command.Call_forest.map txn.account_updates ~f:(fun x ->
+               { (Account_update.forget_aux x) with
+                 Account_update.Poly.authorization =
+                   Mina_base.Control.Poly.Proof
+                     (Lazy.force Proof.blockchain_dummy)
+               } )
+    }
+  in
+
+  (* Print the transaction ID *)
+  let transaction_id =
+    Binable.to_string
+      (module Mina_base.User_command.Stable.Latest)
+      (Zkapp_command txn)
+    |> Base64.encode
+    |> function Ok x -> x | Error _ -> ""
```

Comment:
I'd wrap this whole thing in a `transaction_id_of_zkapp_command` function returning an `option`. 
And here I'd just unwrap the result and let it fail if it's `Error _`.